### DISCUSSION TITLE
Can I expose a custom schema via the API?

### DISCUSSION THREAD

**Discussion by d-mok** [supabase/supabase#1937] [Questions] [answered]
Currently, it is a default that only the 'public' schema is expose via the supabase API. This forces me to put all my tables in the public schema, which is not good for table management. Is it possible for me to create other schemas to store tables, but still be able to expose them via the supabase API?

**Comment by steve-chavez** [comment, marked as answer]
Not for now, but it's a planned enhancement. See this previous discussion: https://github.com/supabase/supabase/discussions/1222

> This forces me to put all my tables in the public schema, which is not good for table management.

You could create a `private` schema and put your tables there. Then only expose views of certain tables on the `public` schema.
